embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/ModuleInitBootstrapper.kt

```diff
@@ -430,4 +461,32 @@ internal class ModuleInitBootstrapper(
     }
 
     private fun isInitialized(): Boolean = asyncInitTask.get() != null
+
+    private fun <T> init(module: KClass<*>, provider: Provider<T>): T =
+        Systrace.traceSynchronous("${moduleNames[module] ?: "module"}-init") { provider() }
+
+    private fun <T> postInit(module: KClass<*>, code: () -> T): T =
+        Systrace.traceSynchronous("${moduleNames[module] ?: "module"}-post-init") { code() }
+
+    companion object {
+        private val moduleNames: Map<KClass<*>, String> = mapOf(
```

Comment:
Could `Class#simpleName` be an alternative to this? That way we don't have to maintain a fairly brittle collection of names in production code

Reply:
Hmm. yeah, I guess obfuscation doesn't matter eh? Let me change that